src/screens/main/ai/chatbot/components/chatCard/ChatCard.tsx

```
@@ -0,0 +1,22 @@
+  text: string;
+  who: "user" | "ai";
+}
+
+export default function ChatCard({ text, who }: ChatCardProps) {
+  return (
+    <S.Container who={who}>
+      {who === "ai" && (
+        <Image source={img_AI_profile_default} style={{ width: 32, height: 32 }} />
+      )}
+      <S.TextBox who={who}>
+        <Typography font="semiBold16" color={who === "user" ? "defaultWhite" : "defaultBlack"} children={text} />
```

Biome (2.4.6)

[error] Avoid passing children using a prop

(lint/correctness/noChildrenProp)